src/sass/_punchcard.scss

```diff
@@ -1,5 +1,32 @@
+////////////////////////
+// Compass Extensions
+@import 'toolkit';
+@import 'toolkit/kickstart';
```

Comment:
Instead of `kickstart`, import `toolkit/reset` _before_ `@import 'toolkit';`.

We want to use a CSS Reset, and that should be the first thing imported